Website/DesktopModules/Hotcakes/Checkout/CheckoutView.ascx.cs

```diff
@@ -25,12 +25,15 @@
 
 using System;
 using Hotcakes.Commerce.Dnn.Web;
+using Hotcakes.Payment;
 
 namespace Hotcakes.Modules.Checkout
 {
     public partial class CheckoutView : HotcakesModuleBase
     {
-        private string _action = "Index";
+        private const string IndexAction = "index";
```

Comment:
The casing of the value changed.  Will this be a breaking change, or does the casing matter at all?

Reply:
Here I have refactored and removed the duplicate code in the variable assignment. The ToLower() method is applied before comparing the strings. In this case, I created the constant "index" already in lowercase